server/routerlicious/packages/services-core/src/delta.ts

```diff
@@ -13,6 +13,7 @@ export interface IDeltaService {
 		from?: number,
 		to?: number,
 		caller?: string,
+		config?: Record<string, any>,
```

Comment:
Can the config pass in as part of the constructor?

Reply:
thats a good suggestion @zhangxin511 , we can do that directly in the FRS fallbackDeltaService constructor.. since thats where we need to use this config. That way, we wont need the OSS changes at all, is that the correct understanding?

Reply:
Just saw your updated comment, that makes sense. I will create a FRS side PR and discard this one. Thanks!